### DISCUSSION TITLE
How to smooth zoom 1 000 000 points using scatter plot in Nextjs

### DISCUSSION THREAD

**Discussion by chiddekel** [chartjs/chartjs-plugin-zoom#870] [Q&A]
??

**Comment by thejoecode** [comment]
Check out the built in Decimation plugin if you meet the requirements: https://www.chartjs.org/docs/latest/configuration/decimation.html#requirements

Another option would be to reduce the number of points at the beginning of the zoom and draw the full display on zoom complete.
